opentelemetry-datadog/examples/agent_sampling.rs

```diff
@@ -66,6 +67,13 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
                 .with_id_generator(RandomIdGenerator::default()),
         )
         .install_simple()?;
+    global::set_tracer_provider(provider.clone());
+    let scope = InstrumentationScope::builder("opentelemetry-datadog")
```

Comment:
opentelemetry-datadog is not the right name. Please include "example/demo" so people don't mistake it!